Source/Particles/Collision/BinaryCollision/NuclearFusion/ProtonBoronFusionInitializeMomentum.H

```diff
@@ -104,7 +104,9 @@ namespace {
             soa_2.m_rdata[PIdx::uz][idx_2], m2,
             ux_alpha1, uy_alpha1, uz_alpha1, static_cast<amrex::ParticleReal>(m_alpha),
             ux_Be, uy_Be, uz_Be, static_cast<amrex::ParticleReal>(m_beryllium),
-            E_fusion, engine);
+            E_fusion,
+            /*isotropic scattering=*/true,
```

Comment:
Based on https://github.com/BLAST-WarpX/warpx/pull/6250#discussion_r2426857062:
```suggestion
            /*isotropic_scattering=*/true,
```